pkg/sql/catalog/systemschema/system.go

```diff
@@ -55,16 +55,20 @@ CREATE TABLE system.descriptor (
   CONSTRAINT "primary" PRIMARY KEY (id)
 );`
 
+	// UsersTableSchema represents the system.users table.
 	UsersTableSchema = `
 CREATE TABLE system.users (
-  username         STRING,
-  "hashedPassword" BYTES,
-  "isRole"         BOOL NOT NULL DEFAULT false,
-	user_id          OID NOT NULL DEFAULT oid(nextval(48:::OID)),
-  CONSTRAINT "primary" PRIMARY KEY (username),
-	INDEX (user_id)
+  username STRING NOT NULL,
+  "hashedPassword" BYTES NULL,
+  "isRole" BOOL NOT NULL DEFAULT false,
+  user_id OID NOT NULL,
+  CONSTRAINT "primary" PRIMARY KEY (username ASC),
+  UNIQUE INDEX users_user_id_idx (user_id ASC),
+  FAMILY "primary" (username),
+  FAMILY "fam_2_hashedPassword" ("hashedPassword"),
+  FAMILY "fam_3_isRole" ("isRole"),
+  FAMILY fam_4_user_id (user_id)
```

Comment:
what's this about? Maintaining backwards compat and making it more obvious what's going on?